src/table/condensed.scss

```diff
@@ -5,8 +5,11 @@
 
 @mixin iui-table-condensed {
   .iui-table-header .iui-cell,
-  .iui-row .iui-cell,
   .iui-paginator {
     min-height: $iui-baseline * 4;
```

Comment:
so header and paginator don't have border?

Reply:
Yup